src/best-practices/reverse-proxies.rst

```diff
@@ -193,31 +193,12 @@ requests from ``http(s)://domain.com/...`` to ``http://localhost:5984/...``
 
     domain.com {
 
-        import /path/to/other/config.caddy # logging, error handling etc.
-
-        proxy / localhost:5984 {
-            transparent
-        }
+       reverse_proxy localhost:5984
 
     }
 
-.. Note::
-    The ``transparent`` preset in the ``proxy`` directive is shorthand for:
-
-    .. code-block:: text
-
-        header_upstream Host {host}
-        header_upstream X-Real-IP {remote}
-        header_upstream X-Forwarded-For {remote}
-        header_upstream X-Forwarded-Proto {scheme}
```

Comment:
Are all of these enabled by default now with the new `reverse_proxy` paragraph?

Reply:
yes

Reply:
actual caddy config is much shorter. 😄 

```
:443
reverse_proxy 127.0.0.1:5984
```